tools/update_versions.py

```diff
@@ -118,8 +132,8 @@ def _bump_release(self, pkg, change_log_header=None, user_name=None,
             new_commit_id = new_source["commit_id"]
             LOG.info("Updating package %s from %s to %s" % (
                 self.pkg.name, old_commit_id, new_commit_id))
-            change_log_lines += _get_git_log(
-                new_source["repo"], old_commit_id)
+            change_log_lines += _get_git_last_commit_log(
+                new_source["repo"], new_commit_id)
```

Comment:
@fabianorosas Ok, that may be a reason to keep the function. Why do we need to remove quotes though?